src/main/java/ee/carlrobert/codegpt/completions/llama/LlamaModel.java

```diff
@@ -113,7 +113,41 @@ public enum LlamaModel {
           HuggingFaceModel.PHI_3_3_8B_4K_Q5_K_S,
           HuggingFaceModel.PHI_3_3_8B_4K_Q6_K,
           HuggingFaceModel.PHI_3_3_8B_4K_Q8_0,
-          HuggingFaceModel.PHI_3_3_8B_4K_FP16));
+          HuggingFaceModel.PHI_3_3_8B_4K_FP16)),
+  CODE_QWEN(
+      "CodeQwen1.5-7B", """
```

Comment:
The "1.5-7B" should be removed from the label, since its redundant in the UI:
![image](https://github.com/carlrobertoh/CodeGPT/assets/39240633/c5e189c9-8aa8-4fbb-837f-310170e633de)